src/lib/validate-forms.js

```diff
@@ -0,0 +1,67 @@
+const api = require('./api')();
+const argsFormFilter = require('./args-form-filter');
+const fs = require('./sync-fs');
+const log = require('./log');
+const { getFormDir, getFormFilePaths } = require('./forms-utils');
+
+//TODO Add this to the main loop
+module.exports = (projectDir, subDirectory, options) => {
+  if (!options) options = {};
```

Comment:
Use param defaults, eg:

```suggestion
module.exports = (projectDir, subDirectory, options={}) => {
```